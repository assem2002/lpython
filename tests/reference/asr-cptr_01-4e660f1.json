{
    "basename": "asr-cptr_01-4e660f1",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/cptr_01.py",
    "infile_hash": "92647f6c175f3599be307df700902b5f7fe69826f22fd5f082713ee4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-cptr_01-4e660f1.stderr",
    "stderr_hash": "8242c95361704d7cd6822c6e71e553700fb2efc45394ffe3e9995b79",
    "returncode": 2
}