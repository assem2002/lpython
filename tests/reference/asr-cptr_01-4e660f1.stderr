semantic error: Indexing CPtr typed expressions is not supported yet
 --> tests/errors/cptr_01.py:5:11
  |
5 |     print(x[0])
  |           ^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
